openqa-investigate

```diff
@@ -24,45 +24,61 @@ jq_output_limit="${jq_output_limit:-15}"
 curl_args=(--user-agent "openqa-investigate")
 echoerr() { echo "$@" >&2; }
 
-clone() {
-    local origin id name_suffix refspec job_data unsupported_cluster_jobs name base_prio clone_settings casedir repo out url clone_id
+declare -a handled_job_ids
+
+restart_job() {
```

Comment:
Despite the name of the API I prefer the word "clone" as we are not actually restarting the original job which is immutable but rather really create a clone and it's actually an altered clone. So I suggest to keep the wording "clone". Maybe we find an even better wording for "clone or copy but with changes" or "create new jobs based on existing ones with only small changes". I would say "clone" does not imply that we would use a clone script or something. Can be a /restart API just as well. One more benefit of "clone" vs. "restart" is just 5 characters vs. 7 ;)

Reply:
I would argue Investigation creates "clones" but also alternate versions of the job. Can we quantify what we think is a clone? What has to be the same?